### DISCUSSION TITLE
Automatic verification of updates using current client similar to Bisq

### DISCUSSION THREAD

**Discussion by modl21** [cryptoadvance/specter-desktop#753] [Specter UI/UX]
Build in an update mechanism which includes verification of the new client similar to Bisq's update process.

Would make the update process much simpler and secure for non technical users since they would only have to do manual signature verification the first time they install specter. The majority of users will still most likely not do manual signature verification on first install unfortunately but it still improves their trust model since all they need to do is download a non malicious client once and they are good.

**Comment by k9ert** [comment]
Isn't that already built into the electron-app @ben-kaufman ? At least we have imho a good foundation for that feature.
And i don't think that make sense for pip installation.
